public/code/person_id_tracker.py

```diff
@@ -0,0 +1,79 @@
+# python script to turn parentID into hierarchicall finesse
+# Made by Decruw Cedric and marc test
+
+# import 
+
+import csv
+import pandas as pd
+# dependencies
+
+path_file = "C:\\Users\\cedric\\Desktop\\new_excel_file_parentID.txt"
+
+# functions
+
+def loopover(parent,hierarchy,parentdiclist):
+    for dic in parentdiclist:
+        if dic['spcolid'] == parent:
+            if dic['parent_spcolid'] != 'NULL':
+                #continue hierarchy
+                hierarchy.append(dic['parent_spcolid'])
+                loopover(dic['parent_spcolid'],hierarchy,parentchilddic)
+
+
+#script
+parentchilddic = []
+spcols = []
+parentspcol = []
+with open(path_file, newline='',) as csvfile:
+    spamreader = csv.reader(csvfile, delimiter='\t', quotechar='|')
+    x = 0
+    for row in spamreader:
+        spcols.append(row[0])
+        parentspcol.append(row[3])
+        parentchilddic.append({'spcolid':row[0],'parent_spcolid':row[3]})
+        x+=1
+        #print(', '.join(row))
```

Comment:
remove not used commented code

Reply:
This is also a file that is used for testing the preview functionality of the webcomponent.
I will solve this if the file is in any other repo.